src/content/en/fundamentals/instant-and-offline/web-storage-overview/index.md

```diff
@@ -0,0 +1,158 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# Web Storage Overiew {: .page-title }
+
+{% include "web/_shared/contributors/mco.html" %}
+
+A good storage engine makes sure your information is saved reliably, reduces bandwidth, and improves responsiveness. The right storage caching strategy is a core building block for enabling offline mobile web experiences. It’s important to choose the right storage mechanisms, both for local device storage and for cloud based server storage. 
+
+This article provides a brief foundation for evaluating storage APIs and services, after which we’ll provide a comparison table and some general guidance. In the near future, we plan to add resources for understanding selected storage topics in greater depth.
+
+# Storage Taxonomy
```

Comment:
There can only be one H1 (the page title) per page. If there are multiples, it breaks the layout. Push all of the headings down one level.